cash-register/index.html

```diff
@@ -0,0 +1,13 @@
+<html>
+<head>
+	<title>Cash Register App!</title>
+</head>
+<body>
+	
+	<script type="text/javascript" src="js/Item.js"></script>
+	<script type="text/javascript" src="js/ItemList.js"></script>
+	<script type="text/javascript" src="js/CashRegister.js"></script>
+	<script type="text/javascript" src="js/app.js"></script>
+
+</body>
+</html>
```

Comment:
@meowbot24 Style - Add a newline at the end of files. The little 🚫⏎ icons here suggests we should add a newline.